eFFORT/hybrid/hybrid_model.py

```diff
@@ -13,7 +15,7 @@ class Hybrid:
 
     """
 
-    def __init__(self, hybrid_config: str = None) -> None:
+    def __init__(self, hybrid_config: Union[str, bytes, PathLike] = None) -> None:
```

Comment:
A nicer option would be `Union[typing.AnyStr, os.PathLike]`.

Reply:
Moritz just told me, that you already wrote that you could have used `AnyStr`. So sorry for missing this.

Reply:
Added commit with change to AnyStr